src/network/dandelion.py

```diff
@@ -47,15 +49,36 @@ def poissonTimeout(start=None, average=0):
             average = FLUFF_TRIGGER_MEAN_DELAY
         return start + expovariate(1.0 / average) + FLUFF_TRIGGER_FIXED_DELAY
 
+    def init_pool(self, pool):
+        """Lazy intialization of pool attr"""
+        self.pool = pool
+
+    def init_is_enabled(self, config):
+        """Lazy intialization of enabled attr"""
```

Comment:
Do we need a separate `init_is_enabled` and `is_dandelion_enabled`?

Reply:
merged it to `is_dandelion_enabled` by changed it into classmethod from staticmethod. and removed `init_is_enabled`.